jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -112,6 +128,14 @@ def _create_viewer_callbacks(self, viewer):
     def _on_viewer_added(self, msg):
         self._create_viewer_callbacks(self.app.get_viewer_by_id(msg.viewer_id))
 
+    def _on_global_display_unit_changed(self, msg):
+        new_unit = u.Unit(msg.unit)
+        uc = self.app._jdaviz_helper.plugins.get("Unit Conversion", None)
+        if new_unit != self.current_unit and new_unit in uc.sb_unit.choices:
```

Comment:
this seems to be the cause of most of the test failures.  `sb_unit` is only exposed for cubeviz (and sometimes specviz) but this if-statement tries to access it regardless.

Reply:
Thank you, @kecnry, for the feedback. Filtering cubeviz fixed 10 of the failing test cases.